data/installed-tests/fwupd.sh

```diff
@@ -27,5 +26,13 @@ run_test synaptics-prometheus-self-test
 run_test dfu-self-test
 run_test mtd-self-test
 
+# grab device tests from the CDN to avoid incrementing the download counter
+export FWUPD_DEVICE_TESTS_BASE_URI=http://cdn.fwupd.org/downloads
+for f in `grep --files-with-matches -r emulation-url @devicetestdir@`; do
+        echo "Emulating for $f"
+        fwupdmgr --force device-test --no-unreported-check --no-remote-check --no-metadata-check "$f"
```

Comment:
why `--force`?

Reply:
It's the only time we look at the emulation-url -- otherwise we can't use the device-test on the actual hardware. Maybe something like --emulate instead perhaps?

Reply:
Yeah I like that better

Reply:
So, actually, mixing the two things together is kinda weird. I think maybe we should have a `device-emulate` *command* that runs the device tests *only* if they have an emulation. It's weird to run `device-test --emulate` and have some emulated, and some run on actual hardware.

Reply:
Or `emulation-device-test` perhaps? Other ideas welcome.

Reply:
device-emulate works well.